tests/pw_parser.spec.ts

```diff
@@ -140,5 +171,5 @@ test('parses test details from the results @unit_test', async ({parsedResults})
   expect(parsedResults.tests[0].endedAt).toEqual('2021-12-18T09:43:23.630Z');
   expect(parsedResults.tests[0].name).toEqual('nested foo > foo - L2 > basic test @broke');
   expect(parsedResults.tests[0].suiteName).toEqual('nested foo > foo - L2');
-  expect(parsedResults.tests[0].browser).toBe(undefined);
+  expect(parsedResults.tests[0].browserCapabilities).toBe(undefined);
```

Comment:
should this still be undefined? it'd be good to have atleast one 1 that covers the browserCapabilities

Reply:
okay, i fix:)